### PR TITLE
[docs] Add information about SIEM on the security page

### PR DESCRIPTION
**Pull request by Zhbert**
## Description

Added information about SIEM.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: docs
type: fix
summary: Added information about SIEM.
impact_level: low
```

### CODE REVIEW

File: docs/documentation/pages/SECURITY_SOFTWARE_SETUP_RU.md [hunk 1]
```diff
@@ -10,9 +10,204 @@ Deckhouse Kubernetes Platform (DKP) использует следующие ди
 
 {% include security_software_setup.liquid %}
 
+## SIEM - ПО для сбора и анализа информации о событиях безопасности
+
+SIEM (Security information and event management) - класс программных продуктов, предназначенных для сбора и анализа информации о событиях безопасности.
```
Comment (z9r5) [inline docs/documentation/pages/SECURITY_SOFTWARE_SETUP_RU.md:15]:
```suggestion
SIEM (Security information and event management) — класс программных продуктов, предназначенных для сбора и анализа информации о событиях безопасности.
```

File: docs/documentation/pages/SECURITY_SOFTWARE_SETUP_RU.md [hunk 2]
```diff
@@ -10,9 +10,204 @@ Deckhouse Kubernetes Platform (DKP) использует следующие ди
...
+SIEM (Security information and event management) — класс программных продуктов, предназначенных для сбора и анализа информации о событиях безопасности.
+
+### KUMA (Kaspersky Unified Monitoring and Analysis Platform)
+
+KUMA объединяет продукты «Лаборатории Касперского» и сторонних поставщиков в единую систему информационной безопасности и является ключевым компонентом на пути реализации комплексного защитного подхода, способного обезопасить от актуальных киберугроз корпоративную и индустриальную среду, а также наиболее эксплуатируемый злоумышленниками стык IT/OT-систем.
+
+#### Описание настроек
+
+{% alert level="warning" %}
+Для работы с KUMA должен быть **обязательно включён** модуль [log-shipper](/modules/log-shipper/).
```
Comment (z9r5) [inline docs/documentation/pages/SECURITY_SOFTWARE_SETUP_RU.md:24]:
```suggestion
Для работы с KUMA должен быть **обязательно включён** модуль [log-shipper](./modules/log-shipper/).
```

File: docs/documentation/pages/SECURITY_SOFTWARE_SETUP_RU.md [hunk 3]
```diff
@@ -10,9 +10,204 @@ Deckhouse Kubernetes Platform (DKP) использует следующие ди
...
+#### Описание настроек
+
+{% alert level="warning" %}
+Для работы с KUMA должен быть **обязательно включён** модуль [log-shipper](./modules/log-shipper/).
+{% endalert %}
+
+Для отправки данных [в KUMA](https://go.kaspersky.com/ru-kuma) необходимо настроить на стороне DKP следующие ресурсы:
+
+- [`ClusterLogDestination`](/modules/log-shipper/cr.html#clusterlogdestination);
+- [`ClusterLoggingConfig`](/modules/log-shipper/cr.html#clusterloggingconfig).
```
Comment (z9r5) [inline docs/documentation/pages/SECURITY_SOFTWARE_SETUP_RU.md:30]:
```suggestion
- [`ClusterLogDestination`](modules/log-shipper/cr.html#clusterlogdestination);
- [`ClusterLoggingConfig`](modules/log-shipper/cr.html#clusterloggingconfig).
```

File: docs/documentation/pages/SECURITY_SOFTWARE_SETUP_RU.md [hunk 4]
```diff
@@ -10,9 +10,204 @@ Deckhouse Kubernetes Platform (DKP) использует следующие ди
...
+SIEM (Security information and event management) — класс программных продуктов, предназначенных для сбора и анализа информации о событиях безопасности.
+
+### KUMA (Kaspersky Unified Monitoring and Analysis Platform)
+
+KUMA объединяет продукты «Лаборатории Касперского» и сторонних поставщиков в единую систему информационной безопасности и является ключевым компонентом на пути реализации комплексного защитного подхода, способного обезопасить от актуальных киберугроз корпоративную и индустриальную среду, а также наиболее эксплуатируемый злоумышленниками стык IT/OT-систем.
+
+#### Описание настроек
+
+{% alert level="warning" %}
+Для работы с KUMA должен быть **обязательно включён** модуль [log-shipper](./modules/log-shipper/).
```
Comment (z9r5) [inline docs/documentation/pages/SECURITY_SOFTWARE_SETUP_RU.md:24]:
```suggestion
Для работы с KUMA должен быть **обязательно включён** модуль [log-shipper](modules/log-shipper/).
```

File: docs/documentation/pages/SECURITY_SOFTWARE_SETUP_RU.md [hunk 5]
```diff
@@ -10,9 +10,204 @@ Deckhouse Kubernetes Platform (DKP) использует следующие ди
...
+    - /var/log/kube-audit/audit.log
+  logFilter:
+    - field: userAgent
+      operator: Regex
+      values: [ "kubelet.*" ]
+  destinationRefs:
+    - kuma-tcp-syslog
+```
+
+#### Отправка логов в Kafka
```
Comment (z9r5) [inline docs/documentation/pages/SECURITY_SOFTWARE_SETUP_RU.md:170]:
```suggestion
#### Отправка логов в Apache Kafka
```

File: docs/documentation/pages/SECURITY_SOFTWARE_SETUP_RU.md [hunk 6]
```diff
@@ -10,9 +10,204 @@ Deckhouse Kubernetes Platform (DKP) использует следующие ди
...
+
+{% alert level="info" %}
+При условии, что Kafka настроена на приём данных.
```
Comment (z9r5) [inline docs/documentation/pages/SECURITY_SOFTWARE_SETUP_RU.md:173]:
```suggestion
При условии, что Apache Kafka настроена на приём данных.
```

File: docs/documentation/pages/SECURITY_SOFTWARE_SETUP_RU.md [hunk 7]
```diff
@@ -10,9 +10,204 @@ Deckhouse Kubernetes Platform (DKP) использует следующие ди
 
 {% include security_software_setup.liquid %}
 
+## SIEM - ПО для сбора и анализа информации о событиях безопасности
```
Comment (z9r5) [inline docs/documentation/pages/SECURITY_SOFTWARE_SETUP_RU.md:13]:
```suggestion
## SIEM (cбор и анализ информации о событиях безопасности)
```

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

File: docs/documentation/pages/SECURITY_SOFTWARE_SETUP_RU.md [hunk 8]
```diff
@@ -10,8 +10,203 @@ Deckhouse Kubernetes Platform (DKP) использует следующие ди
...
+
+Kaspersky Unified Monitoring and Analysis Platform (KUMA) объединяет продукты «Лаборатории Касперского» и сторонних поставщиков в единую систему информационной безопасности и является ключевым компонентом на пути реализации комплексного защитного подхода, способного обезопасить от актуальных киберугроз корпоративную и индустриальную среду, а также наиболее эксплуатируемый злоумышленниками стык IT/OT-систем.
+
+#### Описание настроек
+
+{% alert level="warning" %}
+Для работы с KUMA должен быть **обязательно включён** модуль [log-shipper](modules/log-shipper/).
+{% endalert %}
+
+Для отправки данных в [KUMA](https://go.kaspersky.com/ru-kuma) необходимо настроить на стороне DKP следующие ресурсы:
```
Comment (Zhbert) [inline docs/documentation/pages/SECURITY_SOFTWARE_SETUP_RU.md:27]:
```suggestion
Для отправки данных [в KUMA](https://go.kaspersky.com/ru-kuma) необходимо настроить на стороне DKP следующие ресурсы:
```

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [11075](https://github.com/deckhouse/deckhouse/pull/11075) to the branch [release-1.67](https://github.com/deckhouse/deckhouse/tree/release-1.67) successful!

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [11076](https://github.com/deckhouse/deckhouse/pull/11076) to the branch [release-1.66](https://github.com/deckhouse/deckhouse/tree/release-1.66) successful!

**Pull request merged**